utils/rpms/daos.spec

```diff
@@ -22,7 +22,11 @@ BuildRequires: libpsm2-devel
 BuildRequires: gcc-c++
 BuildRequires: openmpi3-devel
 BuildRequires: hwloc-devel
-BuildRequires: libpsm2-devel
+%if (0%{?rhel} >= 7)
```

Comment:
Why not combine this with the if block on line 30?

Reply:
Thought about that, but saw several combine-able places are not combined, guessed maybe some reason I don't know (sequence, dependency?) behind that, but json-c is very independent so it's safe to combine.